translations/messages.en.yml

```diff
@@ -500,10 +500,10 @@ import:
         description: This importer will import all your wallabag v2 articles. Go to All articles, then, on the export sidebar, click on "JSON". You will have a "All articles.json" file.
     elcurator:
         page_title: 'Import > elCurator'
-        description: 'This importer will import all your elCurator articles. Go to your preferences in your elCurator account and then, export your content. You will have a JSON file.'
```

Comment:
Why?

Reply:
Because the elCurator website is not yet available. So you can't "go to your preferences".